packages/cli/src/commands/test.ts

```diff
@@ -43,6 +43,12 @@ async function storeTest() {
 
 export default class Test extends Command {
   async run() {
+    if (!existsSync(tmpDir)) {
```

Comment:
Why did we stop generating if the `.faststore` folder wasn't there? This seems to be more annoying and error-prone than beneficial to our users.

Reply:
We will copy the Cypress files once again when running the test command 🤝